### ISSUE TITLE
Can't form a clan with a disbanded clan name.

### ISSUE THREAD

**Issue by lambor590** [The-Fireplace-Minecraft-Mods/Clans#273]
**Describe the bug**
Forming a clan with any name, for example `test`, and disbanding that clan, if I want to form a clan with that name, I will get an error or it will reply saying that the name is already taken. Seems like there is some clan registry that persists and is not being removed when is disbanded. 

**To Reproduce**
1. Form a clan.
2. Disband that clan.
3. Form a clan with a disbanded clan name.
4. Will throw an error or it will say that the name is already taken.

**Screenshots**

Error in console when doing step 3.
![b](https://user-images.githubusercontent.com/58952269/184892056-5e0d877d-a7aa-4d67-b754-c3dc0472c30f.png)

Reply when using `clan details test` command after disbanding the clan named `test`.
![a](https://user-images.githubusercontent.com/58952269/184892826-0d193722-7e60-469b-bdb5-c28b49c43662.png)

**Versions:**
 - Minecraft: 1.12.2
 - Mod Version: 2.1.7

**Comment by powerjungle** [comment]
Also when you disband a clan and do `/clan balance` it gives the following exception:

```
[21:39:51] [Server thread/WARN] [minecraft/CommandHandler]: Couldn't process command: clan balance
java.lang.NullPointerException: null
	at the_fireplace.clans.legacy.commands.ClanSubCommand.func_184882_a(ClanSubCommand.java:63) ~[ClanSubCommand.class:?]
	at the_fireplace.clans.legacy.commands.ClanSubCommand.func_184881_a(ClanSubCommand.java:124) ~[ClanSubCommand.class:?]
	at the_fireplace.clans.legacy.commands.CommandClan.func_184881_a(CommandClan.java:228) ~[CommandClan.class:?]
	at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:119) [bj.class:?]
	at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:91) [bj.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:962) [pa.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:941) [pa.class:?]
	at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:37) [la.class:?]
	at net.minecraft.network.play.client.CPacketChatMessage.func_148833_a(SourceFile:9) [la.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_345]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_345]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:723) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_345]
```

**Comment by The-Fireplace** [comment]
Fixed in Clans 2.1.8
